Lecture6/Task1/CollaboratorClasses/Collaborator.java

```diff
@@ -0,0 +1,34 @@
+package CollaboratorClasses;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Collaborator {
+
+    @JsonProperty("login")
+    private String login;
+    @JsonProperty("id")
```

Comment:
property name the same as variable name. @JsonProperty annotation can be omitted

Reply:
fixed